bin/wmagent-mod-config

```diff
@@ -43,9 +43,23 @@ def importConfigTemplate(filename):
     Given filename, load it and grab the configuration object from it
 
     """
-    modSpecs = importlib.machinery.PathFinder().find_spec(filename)
-    mod = modSpecs.loader.load_module("wmcore_config_input")
-    config = getattr(mod, 'config', None)
+
+    cfgBaseName = os.path.basename(filename).replace(".py", "")
+    cfgDirName = os.path.dirname(filename)
+    if not cfgDirName:
+        modSpecs = importlib.machinery.PathFinder().find_spec(cfgBaseName)
+    else:
+        modSpecs = importlib.machinery.PathFinder().find_spec(cfgBaseName, [cfgDirName])
+    try:
+        modRef = modSpecs.loader.load_module(cfgBaseName)
+    except Exception as ex:
+        msg = "Unable to load Configuration File:\n"
```

Comment:
You can probably shrink all these lines into only 2 (or maybe 3). There is no need to make them that short.